src/lib.rs

```diff
@@ -195,17 +274,21 @@ unsafe fn trace_from_process_id(
         .s
         .copy_from_slice(&kernel_logger_name_with_nul[..]);
 
-    let kernel_logger_name_with_nul_pcstr = PCSTR(kernel_logger_name_with_nul.as_ptr());
+    // let kernel_logger_name_with_nul_pcstr = PCSTR(kernel_logger_name_with_nul.as_ptr());
     // Stop an existing session with the kernel logger, if it exists
     // We use a copy of `event_trace_props` since ControlTrace overwrites it
     {
-        let mut event_trace_props_copy = event_trace_props.clone();
-        let control_stop_retcode = ControlTraceA(
-            None,
-            kernel_logger_name_with_nul_pcstr,
-            addr_of_mut!(event_trace_props_copy) as *mut _,
-            EVENT_TRACE_CONTROL_STOP,
-        );
+        let mut event_trace_props_copy = (*event_trace_props).clone();
+        // SAFETY: controlled input.
+        // https://learn.microsoft.com/en-us/windows/win32/api/evntrace/nf-evntrace-controltracea
+        let control_stop_retcode = unsafe {
+            ControlTraceA(
+                None,
+                KERNEL_LOGGER_NAMEA,
```

Comment:
Wondering what the difference between `kernel_logger_name_with_nul_pcstr` and `KERNEL_LOGGER_NAMEA` is